.tekton/label-operator-53op-push.yaml

```diff
@@ -0,0 +1,341 @@
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  annotations:
+    build.appstudio.redhat.com/commit_sha: '{{revision}}'
+    build.appstudio.redhat.com/target_branch: '{{target_branch}}'
+    pipelinesascode.tekton.dev/max-keep-runs: "3"
+    pipelinesascode.tekton.dev/on-event: '[push]'
+    pipelinesascode.tekton.dev/on-target-branch: '[main,master]'
+  creationTimestamp: null
+  labels:
+    appstudio.openshift.io/application: my-app
+    appstudio.openshift.io/component: label-operator-53op
+    pipelines.appstudio.openshift.io/type: build
+  name: label-operator-53op-on-push
+  namespace: yherzog-tenant
+spec:
+  params:
+  - name: dockerfile
+    value: https://raw.githubusercontent.com/yftacherzog/label-operator/main/Dockerfile
```

Comment:
```suggestion
    value: Dockerfile
```

Reply:
(this one is not needed since in push pipelinerun it will be ok to point directly to main branch, but in future it could be blocked by enterprise contract.)